CMakeLists.txt

```diff
@@ -195,6 +197,15 @@ if(TARGET JIT)
                 )
             target_compile_options(JIT INTERFACE ${OpenMP_CXX_FLAGS})
         endif()
+
+        set(VEXCL_OMP_FLAGS "${OpenMP_CXX_FLAGS}")
+        foreach(item ${OpenMP_CXX_LIBRARIES})
+            set(VEXCL_OMP_FLAGS "${VEXCL_OMP_FLAGS} -l${item}")
+        endforeach()
+	message(STATUS "VEXCL_OMP_FLAGS=${VEXCL_OMP_FLAGS}")
```

Comment:
Can you please replace the tab at the beginning of this line with spaces?

Reply:
That was really just a debugging line, I've dropped it (let me know if you do want it)